src/OperatingSystem.php

```diff
@@ -3,9 +3,107 @@
 namespace Laravel\Dusk;
 
 use Illuminate\Support\Str;
+use InvalidArgumentException;
 
 class OperatingSystem
 {
+    /**
+     * List of available Operating System platforms.
+     *
+     * @var array<string, array{slug: string, binary: string, commands: array<int, string>}>
+     */
+    protected static $platforms = [
+        'linux' => [
+            'slug' => 'linux64',
+            'commands' => [
+                '/usr/bin/google-chrome --version',
+                '/usr/bin/chromium-browser --version',
+                '/usr/bin/chromium --version',
+                '/usr/bin/google-chrome-stable --version',
+            ],
+        ],
+        'mac' => [
+            'slug' => 'mac-x64',
+            'commands' => [
+                '/Applications/Google\ Chrome\ for\ Testing.app/Contents/MacOS/Google\ Chrome\ for\ Testing --version',
+                '/Applications/Google\ Chrome.app/Contents/MacOS/Google\ Chrome --version',
+            ],
+        ],
+        'mac-intel' => [
+            'slug' => 'mac-x64',
+            'commands' => [
+                '/Applications/Google\ Chrome\ for\ Testing.app/Contents/MacOS/Google\ Chrome\ for\ Testing --version',
+                '/Applications/Google\ Chrome.app/Contents/MacOS/Google\ Chrome --version',
+            ],
+        ],
+        'mac-arm' => [
+            'slug' => 'mac-arm64',
+            'commands' => [
+                '/Applications/Google\ Chrome\ for\ Testing.app/Contents/MacOS/Google\ Chrome\ for\ Testing --version',
+                '/Applications/Google\ Chrome.app/Contents/MacOS/Google\ Chrome --version',
```

Comment:
Add support for `Chrome for Testing.app`, available from version `113`: https://googlechromelabs.github.io/chrome-for-testing/latest-versions-per-milestone-with-downloads.json